org.gnome.Extensions.json

```diff
@@ -0,0 +1,47 @@
+{
+  "app-id": "org.gnome.Extensions",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.36",
+  "sdk": "org.gnome.Sdk",
+  "command": "gnome-shell-extension-prefs",
+  "finish-args": [
+    "--share=ipc", "--socket=fallback-x11",
+    "--socket=wayland",
+    "--talk-name=org.gnome.Shell.Extensions",
+
+    /* GNOME 3.34 support */
+    "--talk-name=org.gnome.Shell",
+    "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+    "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+  ],
+  "modules": [
+    {
+      "name": "gnome-extensions-app",
+      "buildsystem": "meson",
+      "builddir": true,
+      "subdir": "subprojects/extensions-app",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://gitlab.gnome.org/GNOME/gnome-shell.git",
+          "tag": "3.36.1",
+          "commit": "8fda054dc53e558efa3a6a1e1d75b84bd01c51ee"
+        },
+        {
+          "type": "patch",
```

Comment:
`"type": "patch"` has also `paths` key which takes an array of patches, reducing the boilerplate a bit. JFYI!

Reply:
TIL, thanks! That's really useful.